composer.json

```diff
@@ -18,9 +18,9 @@
     "drush/drush": "^9",
     "nikic/php-parser": "~3",
     "openeuropa/code-review": "~0.3",
+    "openeuropa/drupal-core-require-dev": "~8.6@alpha",
```

Comment:
It's not necessary to specify `@alpha` in this dependency
Could you change by:
`"openeuropa/drupal-core-require-dev": "~8.6",`